### PR TITLE
Fix Set Member/Role Voice Channel Perms

### PR DESCRIPTION
**Pull request by IgnisAlienus**
**Please describe the changes this PR makes and why it should be merged:**

Round 2 of the pull request, not very familiar with github so bare with me. Hopefully this fixes the eslint errors. 

I added meta data to each of the 5 files. I replaced updateOverwrite with permissionOverwrites which changed with Discord JS v12 to v13 in two files.

I added the ability to color the new Action Anchors like the old Anchors could.

Also added the ability to store VC info with how many users are connected to the VC.

I'm sure there's an easier way to have done these updates to fix eslint than a whole new PR, but I've no idea.

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by DetroitWhiskey136** [comment]
you still have linting errors, to fix these you will need to view the error in your editor. this is not a problem with github.

**Comment by IgnisAlienus** [comment]
Yeah, trying to figure it out. I'm new to this.

**Comment by DetroitWhiskey136** [comment]
seems you got it

**Comment by IgnisAlienus** [comment]
hey! I got it, yaaaay

**Review by almeidx — COMMENTED** [review]
There are default actions for creating and jumping to anchors, so we will not be accepting them

**Comment by IgnisAlienus** [comment]
Okay, I've just added the ability to color them in the action list. This gets confusing with the two anchor types, would the mod anchors be going away. The dbm mod anchors aren't supported to be used by default actions that allow anchor jumping and mod actions that allow anchor jumping don't support the new default anchors......it'd seem to me to just support one and dump the other.

I guess the same stance is on store_voice_channel_info_MOD since it's a default action that I've added back an ability to count members in a VC to that used to exist somewhere, because I used to use it.

Those aside, what about the set_member_voice_channel_perms & the set_role_voice_channel_perms fixing the issue with an update from DiscordJSv12 to DiscordJSv13?

**Comment by almeidx** [comment]
> would the mod anchors be going away?

They have been removed from the repository already so, yes

> it'd seem to me to just support one and dump the other.

**Comment by IgnisAlienus** [comment]
Got it. Let me figure out how to remove those ones, I'll just keep them in my personal mods collection. It'll probably be later today when I get back to the house.

**Comment by almeidx** [comment]
You can just delete the files (`git rm <path>`) and commit the changes afterwords

**Comment by IgnisAlienus** [comment]
Done

File: actions/set_member_voice_channel_perms_MOD.js [hunk 1]
```diff
@@ -83,8 +91,8 @@ module.exports = {
     if (!member) return this.callNextAction(cache);
     if (!channel) return this.callNextAction(cache);
 
-    return channel
-      .updateOverwrite(member.id, options)
+    return channel.permissionOverwrites
+      .edit(member.id, options)
```
Comment (almeidx) [inline actions/set_member_voice_channel_perms_MOD.js:95]:
```suggestion
      .edit(member.id, options, { type: 1 })
```

File: actions/set_role_voice_channel_perms_MOD.js [hunk 2]
```diff
@@ -81,9 +89,9 @@ module.exports = {
     options[data.permission] = data.state === '0' ? true : data.state === '2' ? false : null;
 
     if (role && role.id) {
-      if (channel && channel.updateOverwrite) {
-        channel
-          .updateOverwrite(role.id, options)
+      if (channel && channel.permissionOverwrites.edit) {
+        channel.permissionOverwrites
+          .edit(role.id, options)
```
Comment (almeidx) [inline actions/set_role_voice_channel_perms_MOD.js:94]:
```suggestion
          .edit(role.id, options, { type: 0 })
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/set_role_voice_channel_perms_MOD.js [hunk 3]
```diff
@@ -81,9 +89,9 @@ module.exports = {
     options[data.permission] = data.state === '0' ? true : data.state === '2' ? false : null;
 
     if (role && role.id) {
-      if (channel && channel.updateOverwrite) {
-        channel
-          .updateOverwrite(role.id, options)
+      if (channel && channel.permissionOverwrites.edit) {
```
Comment (almeidx) [inline actions/set_role_voice_channel_perms_MOD.js:92]:
```suggestion
      if (channel?.permissionOverwrites.edit) {
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**